Package.swift

```diff
@@ -58,9 +60,9 @@ let package = Package(
         ),
         .target(
             name: "AzureCommunicationChat",
-            dependencies: ["AzureCore"],
+            dependencies: ["AzureCore", "AzureCommunication"],
```

Comment:
Does this actually depend on `AzureCommunication`? I was under the impression it does not. If it does, it is impacted by #686?

Reply:
We do depend on communication - I am rebasing with Josh's latest changes right now, should not impact us though because Chat doesn't use the identifiers he changed but will update accordingly if needed